pkgs/tools/misc/conserver/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, autoreconfHook
+, gssapiSupport ? false
+, libkrb5
+, freeipmiSupport ? false
+, freeipmi
+, ipv6Support ? true
+, opensslSupport ? true
+, openssl
+, trustUdsCredSupport ? false
+, udsSupport ? false
+}:
+
+stdenv.mkDerivation rec {
+  pname = "conserver";
+  version = "8.2.7";
+
+  src = fetchFromGitHub {
+    owner = "bstansell";
+    repo = "conserver";
+    rev = "v${version}";
+    sha256 = "sha256-LiCknqitBoa8E8rNMVgp1004CwkW8G4O5XGKe4NfZI8=";
+  };
+
+  # Remove upon next release since upstream is fixed
+  # https://github.com/bstansell/conserver/pull/82
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/bstansell/conserver/commit/84fc79a459e00dbc87b8cfc943c5045bfcc7aeeb.patch";
+      sha256 = "sha256:1dy8r9z7rv8512fl0rk5gi1vl02hnh7x0i6flvpcc13h6r6fhxyc";
+    })
+  ];
+
+  nativeBuildInputs = [ autoreconfHook ];
+
+  buildInputs = [ ]
+    ++ lib.optionals (freeipmiSupport) [ freeipmi ]
+    ++ lib.optionals (gssapiSupport) [ libkrb5 ]
+    ++ lib.optionals (opensslSupport) [ openssl ];
+
+  configureFlags = [ "--with-ccffile=/dev/null" "--with-cffile=/dev/null" ]
+    ++ lib.optionals (freeipmiSupport) [ "--with-freeipmi=${freeipmi}/include" ]
+    ++ lib.optionals (gssapiSupport) [ "--with-gssapi=${libkrb5.dev}/include" ]
+    ++ lib.optionals (ipv6Support) [ "--with-ipv6" ]
+    ++ lib.optionals (opensslSupport) [ "--with-openssl=${openssl.dev}/include" ]
+    ++ lib.optionals (trustUdsCredSupport) [ "--with-trust-uds-cred" ]
+    ++ lib.optionals (udsSupport) [ "--with-uds" ];
+
+  # Disabled due to exist upstream cases failing 8/15 tests
```

Comment:
Are they all tests that require network and therefore do not work in our sandbox?

Reply:
It could be the network, but Im not sure. I have some specific results called out in the above comment: https://github.com/NixOS/nixpkgs/pull/181543#issuecomment-1186350878

It is doing things with loopback which might be why the behavior differs in the sandbox? https://github.com/bstansell/conserver/blob/master/test/dotest#L29

Open to ideas about how to test this but wasnt sure how to drop into the tests as they are running during the `nix-build`